InvenTree/part/admin.py

```diff
@@ -122,9 +122,9 @@ class Meta(PartResource.Meta):
         ]
 
 
-class StocktakeInline(admin.TabularInline):
-    """Inline for part stocktake data"""
-    model = models.PartStocktake
```

Comment:
Question: Is there no need for the old inline anymore?

Reply:
IMO no, it has been outdated since the recent stocktake updates